src/applications/mhv-secure-messaging/tests/e2e/secure-messaging-recipients-grouping-compose.cypress.spec.js

```diff
@@ -2,35 +2,61 @@ import SecureMessagingSite from './sm_site/SecureMessagingSite';
 import PatientInboxPage from './pages/PatientInboxPage';
 import GeneralFunctionsPage from './pages/GeneralFunctionsPage';
 import PatientComposePage from './pages/PatientComposePage';
-import { AXE_CONTEXT, Locators, Data } from './utils/constants';
+import PatientInterstitialPage from './pages/PatientInterstitialPage';
+import { AXE_CONTEXT, Locators } from './utils/constants';
 
 describe('SM RECIPIENTS GROUPING ON COMPOSE', () => {
   const updatedFeatureToggles = GeneralFunctionsPage.updateFeatureToggles([
     {
       name: `mhv_secure_messaging_recipient_opt_groups`,
       value: true,
     },
+    {
+      name: `mhv_secure_messaging_curated_list_flow`,
```

Comment:
although it may not be used consistently throughout cypress tests, I would suggest importing toggle key values from `import FEATURE_FLAG_NAMES from '@department-of-veterans-affairs/platform-utilities/featureFlagNames';`